agent/format-2.0.go

```diff
@@ -129,6 +130,15 @@ func (formatter_2_0) unmarshal(data []byte) (*configInternal, error) {
 			format.APIPassword,
 		}
 	}
+
+	if format.MongoMemoryProfile != "" {
+		config.mongoMemoryProfile = format.MongoMemoryProfile
+	}
+	if format.JujuDBSnapChannel != "" {
+		config.jujuDBSnapChannel = format.JujuDBSnapChannel
+	}
```

Comment:
Was there a reason to move these, or just so that state-serving info was last?

Reply:
Just to put info last (seen similar elsewhere and my inner organiser wanted things all the same aha)